### PR TITLE
add basics for system_summary api

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: forch/forchestrator.py [hunk 1]
```diff
@@ -92,15 +97,69 @@ def _get_peer_controller_url(self):
...
+            'state_summary_sources': state_summary,
+            'site_name': self._config['site']['name'],
+            'controller_hostname': os.getenv('HOSTNAME')
         }
-        overview.update(self._faucet_collector.get_controller_state())
+        overview.update(self._distill_summary(state_summary))
         return overview
 
+    def _distill_summary(self, summary):
+        try:
```
Comment (anurag6) [inline forch/forchestrator.py]: Lint might crib because of no docstring?

Comment (grafnu) [reply forch/forchestrator.py]: ah, but you don't need docstring for private methods!

File: forch/forchestrator.py [hunk 2]
```diff
@@ -92,15 +97,69 @@ def _get_peer_controller_url(self):
...
+            state_summary['state_summary'] = summary
+            state_summary['state_summary_detail'] = detail
+        except Exception as e:
+            state_summary.update({
+                'state_summary': 'error',
+                'state_summary_detail': str(e)
+            })
+        return state_summary
+
+    def _summary_of_summary(self, summary):
```
Comment (anurag6) [inline forch/forchestrator.py]: Would get_subsystem_state be a better name?

Comment (grafnu) [reply forch/forchestrator.py]: Changed to _get_combined_summary()

File: forch/local_state_collector.py [hunk 3]
```diff
@@ -21,26 +21,36 @@ def __init__(self):
                               'forch':      ('python', r'forchestrator\.py'),
                               'bosun':      ('dunsel_watcher', r'bosun')}
 
-    def get_process_state(self, extended=True):
+    def get_process_summary(self):
```
Comment (anurag6) [inline forch/local_state_collector.py]: Maybe a docstring here then.

Comment (grafnu) [reply forch/local_state_collector.py]: Added -- and elsewhere too!

**Pull request merged**